editor/src/components/canvas/commands/push-intended-bounds-and-update-hugging-elements-command.ts

```diff
@@ -128,39 +133,8 @@ function applyUpdateResizeHuggingElementsCommands(
           'height',
           frameAndTarget.frame.height,
         ),
+        showToastCommand('Added fixed width and height', 'NOTICE', 'added-width-height'), // TODO before merge verify this toast shows up
```

Comment:
🥁

Reply:
I knew there was something I had to check...

Reply:
TODO removed in c3ea88573.